src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -242,13 +243,14 @@ private sealed class InvokeMethod : SourceDelegateMethodSymbol
                 DiagnosticBag diagnostics)
                 : base(delegateType, returnType, syntax, MethodKind.DelegateInvoke, DeclarationModifiers.Virtual | DeclarationModifiers.Public)
             {
-                this.refKind = refKind;
+                this._refKind = refKind;
 
                 SyntaxToken arglistToken;
                 var parameters = ParameterHelpers.MakeParameters(
                     binder, this, syntax.ParameterList, out arglistToken,
                     allowRefOrOut: true,
                     allowThis: false,
+                    addIsConstModifier: true,
```

Comment:
Fixed and modified tests
C# `IsConstIsWrittenOnRefReadOnlyMembers_Delegates_*`
VB `UsingLambdasOfRefReadOnlyDelegatesIsNotSupported_*`

---
In reply to: [119757863](https://github.com/dotnet/roslyn/pull/19658#discussion_r119757863) [](ancestors = 119757863)